test/Microsoft.Azure.Devices.Edge.Test.Common/certs/Certificates.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Certs
+{
+    public class Certificates
+    {
+        private string[] GetFileLocation(string deviceId)
+        {
+            return new[]
+            {
+                $"certs/iot-device-{deviceId}-full-chain.cert.pem",
+                $"private/iot-device-{deviceId}.key.pem"
+            };
```

Comment:
The code today consolidates all hard-coded certificate paths into one file: Common/OsPlatform.cs. It would be nice to preserve that. In this PR, there are hard-coded cert paths in at least three files.

Reply:
The hard-coded value have been centralized to `FixedPaths.cs`